external/Makefile

```diff
@@ -124,7 +139,7 @@ endif
 endif
 
 custom_clean:
-	$(foreach OBJFILE, $(OBJS), rm -f $(OBJFILE))
+	$(foreach OBJFILE, $(OBJS) $(LIBBUNDLED) $(LIBEXTERNAL), rm -f $(OBJFILE))
 
 clean: custom_clean iotivity_clean
 	$(call DELFILE, $(BIN))
```

Comment:
What is this $(BIN) here?

Reply:
Please reply to this commit.

Reply:
The BIN is no more.

Do not get why you think there is a discrepancy between commit message and the contents.
The *clean targets are supposed to remove intermediate artifacts (files and directories, if applicable).
Since this patch introduces a new artifact (libbundled), it needs to be added to clean list.